tests/update_workflow_test.go

```diff
@@ -662,6 +662,7 @@ func (s *functionalSuite) TestUpdateWorkflow_NormalScheduledWorkflowTask_AcceptC
 				case 1:
 					return nil, nil
 				case 2:
+					s.Require().True(len(task.Messages) > 0, "Task has no messages", task)
```

Comment:
What is that? Why not `s.True()`?

Reply:
Without the require we'd immediately access the first element of a zero-length slice on the next line, crashing and dumping a useless stack trace

Reply:
It should always have them, but ok. I will check it later.